permissions/ManagePages.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2020 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\custom_pages\permissions;
+
+use Yii;
+use humhub\modules\admin\components\BaseAdminPermission;
+use humhub\modules\user\models\User;
+
+/**
+ * ManagePages Permissions
+ */
+class ManagePages extends BaseAdminPermission
+{
+
+    /**
+     * @inheritdoc
+     */
+    public $defaultAllowedGroups = [
+        User::USERGROUP_FRIEND,
```

Comment:
Is `User::USERGROUP_FRIEND` really needed here? To be honest this should be under `$fixedGroups`

Reply:
Good catch.

@yurabakhtin  The permission should only be used on a global level (groups) - so probably we can remove the "UserGroups" information here or?

FYI: I created an new Issue/RFE to have a permission on space/user profile level to have ManagePages permission there as well. https://github.com/humhub/humhub-modules-custom-pages/issues/159

Reply:
@luke- 

> The permission should only be used on a global level (groups) - so probably we can remove the "UserGroups" information here or?

Yes, you are right, the permission is used only for user groups, it is not used for Content Container(Space/User) rights, so we don't need to specify the default groups at all there. I fixed it here https://github.com/humhub/humhub-modules-custom-pages/pull/156/commits/5f67579bbe3132b172e0dc086754398ad4113b28.